src/components/Tabs/index.tsx

```diff
@@ -0,0 +1,61 @@
+import styled from '@emotion/styled'
+import { HTMLAttributes, useMemo } from 'react'
+import Tab from './Tab'
+import TabMenu from './TabMenu'
+import TabMenuItem from './TabMenuItem'
+import TabsContext from './TabsContext'
+
+const TabsContainer = styled.ul`
+  display: flex;
+  flex-wrap: nowrap;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  padding-inline-start: 0;
+  overflow-x: scroll;
+  position: relative;
+  z-index: 0;
+  list-style-type: none;
+
+  &::after {
+    z-index: -1;
```

Comment:
Why using z-index ? Maybe put it into `::before` ?

Reply:
Doesn't work too need z-index 😞